webknossos/webknossos/skeleton/skeleton.py

```diff
@@ -7,39 +7,36 @@
 
 from webknossos.utils import warn_deprecated
 
-from .graph import Graph
 from .group import Group
 
 Vector3 = Tuple[float, float, float]
 
-GroupOrGraph = Union[Group, Graph]
-
 
 @attr.define()
 class Skeleton(Group):
     """
-    Representation of the [skeleton tracing](/webknossos/skeleton_annotation.html) of an `Annotation`.
-    It contains metadata to identify the related dataset and is the root-group of sub-groups and graphs.
-    See the parent class `Group` for methods about group and graph handling.
+    Representation of the [skeleton](/webknossos/skeleton_annotation.html) of an `Annotation`.
+    It contains metadata to identify the related dataset and is the root-group of sub-groups and trees.
+    See the parent class `Group` for methods about group and tree handling.
     To upload a skeleton to webknossos, please create an `Annotation()` with it.
 
     A small usage example:
 
     ```python
-    annotation = Annotation("my_annotation", scale=(11, 11, 24))
+    annotation = Annotation("my_annotation", voxel_size=(11, 11, 24))
     group = annotation.skeleton.add_group("a group")
-    graph = group.add_graph("a graph")
-    node_1 = graph.add_node(position=(0, 0, 0), comment="node 1")
-    node_2 = graph.add_node(position=(100, 100, 100), comment="node 2")
+    tree = group.add_tree("a tree")
+    node_1 = tree.add_node(position=(0, 0, 0), comment="node 1")
+    node_2 = tree.add_node(position=(100, 100, 100), comment="node 2")
 
-    graph.add_edge(node_1, node_2)
+    tree.add_edge(node_1, node_2)
     ```
 
     Also see [this example](/webknossos-py/examples/skeleton_synapse_candidates.html) for a more
     complex interaction.
     """
 
-    scale: Vector3
+    voxel_size: Vector3
```

Comment:
Add `scale` property with deprecation warning?

Reply:
This is slightly complicated, since the scale currently is non-optional. I'll add a proposal and we'll see if it's worth it :+1:

Reply:
Sorry, this is just about the property, not about the init-argument, right? Good idea, I'll add this!